dz/index.js

```diff
@@ -0,0 +1,48 @@
+const agregateUsersAndTodos = async(user) => {
+   const url = 'https://jsonplaceholder.typicode.com/users';
+   const todosUrl = 'https://jsonplaceholder.typicode.com/todos'
+   try {
+     const res = await fetch(url).then(Response => Response.json())
+     const todosRes = await fetch(todosUrl).then(Response => Response.json())
+     // console.log(res,todosRes);
+     const aggregate = (users,todoses) => {
+       
+      const aggregate2 = todoses.reduce ((result, todos) => 
+      {
+      result [todos.userId]= todos;
+        return result;},{});
+        return users.map ((user) => (
+          {...user, 
+         todos:aggregate2[user.id],}));
+        }
+    console.log(aggregate(res, todosRes));
+  }
+  catch (error) {
+      
+      }}
+      console.log(agregateUsersAndTodos())
```

Comment:
seems that your function return wrong result. Compare result from function with filter, the are quite different.